test/test_native_transfer.py

```diff
@@ -0,0 +1,59 @@
+from gql import gql
+from functools import reduce
+import time, unittest, base
+
+
+class TestTransfer(base.Base):
+    amount = 5000000
+    denom = "atestfet"
+    msg_type = '/cosmos.bank.v1beta1.MsgSend'
+    db_query = 'SELECT * from native_transfers'
+
+    def test_native_transfer(self):
+        self.db_cursor.execute('TRUNCATE table native_transfers')
+        self.db.commit()
+        self.assertFalse(self.db_cursor.execute(self.db_query).fetchall(), "\nError: table not empty after truncation")
```

Comment:
I think we can factor this out to a method on `base.Base` which takes the table name and the DB query. I think this would improve maintainability and I could also imagine having a collection of similar functions that do different kinds of setup / cleanup.

Reply:
What do you think about also moving this into the `setUpClass` method instead? We can't use the unittest assertions because we won't have a TestCase instance [but we can `Raise` instead](https://github.com/fetchai/ledger-subquery/blob/main/test/test_native_primitives.py#L42).

Reply:
What do you think about also moving the call to that method into the `setUpClass` method instead? We can't use the unittest assertions because we won't have a TestCase instance [but we can `Raise` instead](https://github.com/fetchai/ledger-subquery/blob/main/test/test_native_primitives.py#L36).

Reply:
What do you think about also moving the call to that method into the `setUpClass` method instead of in the test? We can't use the unittest assertions because we won't have a TestCase instance [but we can `Raise` instead](https://github.com/fetchai/ledger-subquery/blob/main/test/test_native_primitives.py#L36).